composer.json

```diff
@@ -14,7 +14,7 @@
 		"php-parallel-lint/php-parallel-lint": "1.3.2",
 		"sirbrillig/phpcs-changed": "2.10.0",
 		"squizlabs/php_codesniffer": "^3.6.2",
-		"wp-coding-standards/wpcs": "2022-02-07 as 2.3.1"
+		"wp-coding-standards/wpcs": "2023-01-04 as 2.3.1"
```

Comment:
Make sure this is something you want to do

Reply:
Git pull did something weird.  I did a manual rebase and it fixed it.